### PR TITLE
Replaced Add note icon with new Add note icon

### PR DESCRIPTION
**Pull request by prathamsoni11**
## Purpose / Description
Added new icon for Add note as mentioned in issue #10773 .

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ ] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

Screenshot_2022-07-25-14-53-04-913_com ichi2 anki
 any new or changed strings)

### CODE REVIEW

File: AnkiDroid/src/main/res/drawable/ic_add_note.xml [hunk 1]
```diff
@@ -25,6 +25,6 @@
     android:viewportWidth="24"
     android:viewportHeight="24">
   <path
-      android:pathData="M18.3711,6.2324C17.3254,6.2419 16.1991,6.2498 15.7422,6.2441L4.4961,6.2441C3.3961,6.2441 2.4961,7.1441 2.4961,8.2441L2.4961,16.2441C2.4961,17.3441 3.3961,18.2441 4.4961,18.2441L18.4961,18.2441C19.5961,18.2441 20.4961,17.3441 20.4961,16.2441C20.4338,14.8996 20.4687,10.7235 20.502,8.1699C20.5161,7.0871 19.5684,6.2216 18.3711,6.2324zM6.9805,8.3496L8.9805,8.3496L8.9805,11.1484L11.7793,11.1484L11.7793,13.1484L8.9805,13.1484L8.9805,15.9492L6.9805,15.9492L6.9805,13.1484L4.2793,13.1484L4.2793,11.1484L6.9805,11.1484L6.9805,8.3496z"
-      android:fillColor="#ffffff"/>
+      android:pathData="M14,2L6,2c-1.1,0 -1.99,0.9 -1.99,2L4,20c0,1.1 0.89,2 1.99,2L18,22c1.1,0 2,-0.9 2,-2L20,8l-6,-6zM16,16h-3v3h-2v-3L8,16v-2h3v-3h2v3h3v2zM13,9L13,3.5L18.5,9L13,9z"
+      android:fillColor="#fff"/>
```
Comment (prateek-singh-3212) [inline AnkiDroid/src/main/res/drawable/ic_add_note.xml:29]:
```suggestion
      android:fillColor="#ffffff"/>
```

**Review by prateek-singh-3212 — COMMENTED** [review]
Looks good to me. Let's rerun the tests @david-allison Can you rerun the tests?

**Comment by prateek-singh-3212** [comment]
@prathamsoni11 Thanks for contributing. Please fill out the complete PR Template

**Comment by BrayanDSO** [comment]
Duplicate of #11915

**Comment by prateek-singh-3212** [comment]
Thanks for pointing it out.

**Comment by mikehardy** [comment]
I apologize this has sat as long as it has, and I really appreciate the attempt to make a change here in AnkiDroid - unfortunately we (maintainers) let the issue of whether to change the icon or not sit on the original issue without really  being clear.

Thinking about it now, I must admit I also prefer the current icon that is the same general card shape as the deck one, so I'm going to close this with apologies

**Pull request closed without merging**